operations/rack_resiliency/rack_resiliency.md

```diff
@@ -0,0 +1,551 @@
+* Ensures load balancing and high availability of critical services essential for the seamless execution and completion of user jobs.
+* Monitors and reports failures of management nodes and racks which are part of the MPFD.
+* Monitors and reports critical services health and their balance across racks.
+* Aligns with future plans for similar functionality in next generation systems management solutions
+
+## Key Terminology
+* Placement: Arrangement of Management Nodes across racks. Note that zone awareness primitives in `Kubernetes` refer to this as "topology".
+* Failure Domain: Failure domains are zones which includes infrastructure that provides availability for CSM services.
+* Zone: A zone represents a logical failure domain. It is common for `Kubernetes` clusters to span multiple zones for increased availability. 
+* `Kubernetes` topology zones: You can use topology spread constraints to control how pods are spread across your cluster among failure-domains such
+  as regions, zones, nodes, and other user-defined topology domains. This can help to achieve high availability as well as efficient resource utilization.
+* Ceph: `Ceph` is the utility storage platform that is used to enable pods to store persistent data. It is deployed to provide block, object, and file storage
+  to the management services running on Kubernetes, as well as for telemetry data coming from the compute nodes.
+* CFS: Configuration Framework Service (CFS) is available on systems for remote execution and configuration management of nodes and boot images.
+  CFS configures nodes and images via a GitOps methodology. All configuration content is stored in the Version Control Service (VCS) and is
+  managed by authorized Infrastructure Administrators.
+
+## Critical services for RR
+
+We define critical services as those services that are critical to execution of user jobs. These services are critical because the user plane / jobs require
+a timely response from these critical services for continued operations. Therefore, in the context of a rack failure, it is crucial that such critical services 
+have replicas and resiliency across racks.
+
+### 1. Categories of CSM critical services
+
+CSM services can be grouped into different categories based on the nature of support they provide to run the user jobs.
+
+#### 1.1 CSM platform services
+
+These services host the platform on which the CSM runs and provide the infra structure for running those services which are critical for user jobs.
+
+    * Example: BOS, BSS, CFS, gitea, IMS, iPXE etc.,
+
+#### 1.2. CSM critical services
+
+These services directly impact the way user jobs are executed and the availability of these services is very much essential for succeeding end user jobs.
+Critical services can not execute without the platform services. So there is dependency on the platform services for critical services to be healthy.

```

Comment:
You mention platform services here but never specify what those are or what that means.